packages/odyssey-storybook/src/components/odyssey-mui/Form/Form.stories.tsx

```diff
@@ -255,6 +243,19 @@ export const Alert: StoryObj<FormProps> = {
   },
 };
 
+export const FullWidth: StoryObj<FormProps> = {
+  ...Template,
+  args: {
+    isFullWidth: true,
+    children: (
+      <>
+        <TextField isFullWidth label="Vessel name" />
+        <TextField isFullWidth isMultiline label="Reason for visit" />
+      </>
+    ),
+  },
+};
```

Comment:
If we're gonna write new stories, we should do it in CSF3 format. With that, we'd remove `...Template` as it doesn't add any necessary props anymore.